src/components/Table.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export const Table = ({ data }) => {
```

Comment:
This is not used. But what is is suppose to do. Seems like it does the same as `CustomizedTables`

Reply:
Yes I dont use it anymore.